suite/tests/samples/memtrace_simple_repstr.templatex

```diff
@@ -1,6 +1,7 @@
 // Without emulation awareness, this will print "1, movs" instead of "2, rep movs".
 Format: <data address>: <data size>, <\(r\)ead/\(w\)rite/opcode>
 0x[0-9a-f]*:  2, rep movs
+0x[0-9a-f]*:  2, rep movs
```

Comment:
Note the instruction entry w/o any load/store entries.  This tracer skips instruction fetches and the instruction entry is supposed to just tell you who did each load/store: so maybe this is undesirable.  We'd have the same thing for scatter/gather with zero actual loads/stores.  I would say let's just live with it for now as it does not seem trivial to avoid: today the static IR query "does this have memory operands" is what's used.

Reply:
> I would say let's just live with it for now as it does not seem trivial to avoid: today the static IR query "does this have memory operands" is what's used.

I think we should document that `memtrace_simple` outputs memory instrs followed by load/stores **if any**. I believe this is better than completely omitting all such mem instrs (ones without any actual load/store) from the trace. This way we have more information in the trace, and a user can do required post-processing to remove what they don't want.

Reply:
+1.  Done.